tieredhashing/tiered_hashing.go

```diff
@@ -421,7 +443,7 @@ func (t *TieredHashing) isCorrectnessPolicyEligible(perf *NodePerf) (float64, bo
 	// should satisfy a certain minimum percentage
 	pct := totalSuccess / perf.NCorrectnessDigest * 100
 
-	return pct, pct >= t.cfg.CorrectnessPct
+	return pct, (t.AverageCorrectnessPct - pct) < t.cfg.CorrectnessThreshold
```

Comment:
is this backwards? ( lets put in a comment of what the boolean means)

if average correctness is 80%
this node correctness is 20%

0.8 - 0.2 = 0.6
0.6 is more than the 20% threshold.

Reply:
Faceplam. You are right.

Reply:
@willscott It is correct actually. This boolean is true when we do NOT want to remove a node from the pool i.e. true means node is eligible for the correctness policy.

Reply:
In your example, bool will be false and so node will be kicked out.

Reply:
can you put in the comment explaining this?